### PR TITLE
[CELEBORN-1531][FOLLOWUP] Assign the scheduled check task

### PR DESCRIPTION
**Pull request by turboFei**
### What changes were proposed in this pull request?

As title.

### Why are the changes needed?
Followup for https://github.com/apache/celeborn/pull/2653

`checkForUnavailableWorkerTimeOutTask` and `checkForS3RemnantDirsTimeOutTask` are not assigned and always null.
image

Combine the `checkForHDFSRemnantDirsTimeOutTask` and `checkForS3RemnantDirsTimeOutTask` with `checkForDFSRemnantDirsTimeOutTask`.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?
GA.

### CODE REVIEW

File: master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala [hunk 1]
```diff
@@ -172,7 +172,6 @@ private[celeborn] class Master(
   private var checkForApplicationTimeOutTask: ScheduledFuture[_] = _
   private var checkForUnavailableWorkerTimeOutTask: ScheduledFuture[_] = _
   private var checkForHDFSRemnantDirsTimeOutTask: ScheduledFuture[_] = _
-  private var checkForS3RemnantDirsTimeOutTask: ScheduledFuture[_] = _
```
Comment (turboFei) [inline master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala:175]: This is not needed anymore, as the task was combined for `hasHDFSStorage` and `hasS3Storage`

File: master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala [hunk 2]
```diff
@@ -171,8 +171,7 @@ private[celeborn] class Master(
   private var checkForWorkerTimeOutTask: ScheduledFuture[_] = _
   private var checkForApplicationTimeOutTask: ScheduledFuture[_] = _
   private var checkForUnavailableWorkerTimeOutTask: ScheduledFuture[_] = _
-  private var checkForHDFSRemnantDirsTimeOutTask: ScheduledFuture[_] = _
-  private var checkForS3RemnantDirsTimeOutTask: ScheduledFuture[_] = _
+  private var checkForDFSRemnantDirsTimeOutTask: ScheduledFuture[_] = _
```
Comment (turboFei) [inline master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala:174]: HDFS + S3 => DFS

**Review by RexXiong — APPROVED** [review]
LGTM

**Comment by turboFei** [comment]
thanks

**Pull request closed without merging**